### ISSUE TITLE
[TRACKING] Live USBs failing self-check

### ISSUE THREAD

**Issue by ct-martin** [RITlug/TigerOS#87]
As mentioned in the meeting, I've observed issues when building ISOs where the boot media is unusable. There has not been consistency in the building, copying, writing, or any other aspect of the ISO building. Upon re-running one of those steps after the issue is detected, it works perfectly fine with no issues (so seems to be random chance?). This issue is to track successful/failed builds. Helpful information includes:
- OS (Stock Fedora, Windows, TigerOS, Arch, etc.)
- How the ISO was built (`./make-iso.sh`, dedicated TigerOS CI (as specified on the wiki), custom CI, etc.)
- TigerOS Revision (branch, latest commit preferred)
- Was it built on your local machine, or was it copied (and how: network, to USB drive, etc.)
- What software was used to write it (Fedora Media Writer, Etcher, Win32DiskImager, Rufus, etc.)

This issue will be updated as needed.

**Comment by jflory7** [comment]
_Discussed in our [weekly meeting](https://github.com/RITlug/TigerOS/wiki/Next-meeting-agenda)._

## Blocked by…

This ticket is blocked by the CI environment not generating ISOs to run an install with the USB media.

## Tracker ticket: Update with changes

This ticket is low priority for now, but if anyone runs into a new issue or has feedback, **add a comment** to this ticket and **update the tags** as critical priority and as a meeting ticket. Otherwise, we'll cover this ticket on an "as needed" basis.

**Comment by Tjzabel** [comment]
I would love to get more information on this issue since we now have a more updated ISO release. It is possible this issue does not need to be here, since we have not had any problems with using USB's in quite a while

**Comment by ct-martin** [comment]
@Tjzabel We also haven't been building \*anything* in a while. I think that if we don't have any issues before we're ready for a stable, push-to-master release then we can close it, but I'd like to let it hover for a little longer.

**Comment by ct-martin** [comment]
Official meeting statement: Will be closed if no issues before first release. May vote at end of semester if frequent building and this has not been an issue.

**Comment by Tjzabel** [comment]
We have had no issues with this occurring.
